packages/block-library/src/navigation-link/edit.js

```diff
@@ -187,6 +187,7 @@ function Controls( { attributes, setAttributes, setIsLabelFieldFocused } ) {
 					url: '',
 					description: '',
 					rel: '',
+					opensInNewTab: false,
```

Comment:
Good catch 👍 

The issue is that this approach is already not sustainable. Any other props that are added probably will get missed from this list and the same bug will reoccur.

Not blocking on this PR landing but something to consider.